### PR TITLE
docs: make enableWebviewDetailsCollection true by default

### PR DESCRIPTION
**Pull request by mwakizaka**
Should be merged after https://github.com/appium/appium-android-driver/pull/699 is merged

https://github.com/appium/appium-android-driver/pull/699/files#r655874047

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Capability Name | Description
 appium:autoWebview | If set to `true` then UiAutomator2 driver will try to switch to the first available web view after the session is started. `false` by default.
 appium:webviewDevtoolsPort | The local port number to use for devtools communication. By default the first free port from 10900..11000 range is selected. Consider setting the custom value if you are running parallel tests.
 appium:ensureWebviewsHavePages | Whether to skip web views that have no pages from being shown in `getContexts` output. The driver uses devtools connection to retrieve the information about existing pages. `true` by default.
-appium:enableWebviewDetailsCollection | Whether to retrieve extended web views information using devtools protocol. Enabling this capability helps to detect the necessary chromedriver version more precisely. `false` by default.
+appium:enableWebviewDetailsCollection | Whether to retrieve extended web views information using devtools protocol. Enabling this capability helps to detect the necessary chromedriver version more precisely. `true` by default.
```
Comment (mykola-mokhnach) [inline README.md:147]: https://github.com/appium/appium-espresso-driver/pull/677#discussion_r656750124

**Comment by mykola-mokhnach** [comment]
it would also make sense to merge this pr as soon as we publish the downstream change to appium-android-driver, so its minimum version could be also bumped in the current package.json

File: README.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Capability Name | Description
 --- | ---
 appium:autoWebview | If set to `true` then UiAutomator2 driver will try to switch to the first available web view after the session is started. `false` by default.
 appium:webviewDevtoolsPort | The local port number to use for devtools communication. By default the first free port from 10900..11000 range is selected. Consider setting the custom value if you are running parallel tests.
-appium:ensureWebviewsHavePages | Whether to skip web views that have no pages from being shown in `getContexts` output. The driver uses devtools connection to retrieve the information about existing pages. `true` by default.
+appium:ensureWebviewsHavePages | Whether to skip web views that have no pages from being shown in `getContexts` output. The driver uses devtools connection to retrieve the information about existing pages. `true` by default since Appium 1.19.0, `false` lower than 1.19.0.
```
Comment (mwakizaka) [inline README.md:146]: I've just happened to see https://github.com/appium/appium/blob/master/CHANGELOG.md#changes-in-version-1190-from-1183

File: README.md (hunk 1 above)
Comment (mwakizaka) [reply README.md]: Thanks. Addressed

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**